### DISCUSSION TITLE
【委托】怎样才能快速的背完一篇课文啊

### DISCUSSION THREAD

**Discussion by sensei-is-playing-GenShin** [share121/inter-knot#4672] [委托]
下午英语老师就要查我的英语背诵了，还没背，怎么办。如果没被骂的话给多少丁尼都行，不，要多少原石多少青辉石都可以。求有用的办法

**Comment by sensei-is-playing-GenShin** [comment]
队友呢，快救一下，要死了

**Comment by sensei-is-playing-GenShin** [comment]
不用救了，死了

**Reply by The-Place-Where-Winter-Sieeps** [reply]
额。。。抱一下吧<img src="https://upload-bbs.miyoushe.com/upload/2023/01/21/860c72d5521a0647514c324d16c7db1b_8005402887548479862.png" alt="爱莉希雅_安慰" width="75" >
